docs/latex/Inhalt/03_Inhalt/01_einleitung.tex

```diff
@@ -0,0 +1,16 @@
+\chapter{Einleitung}
+Im Internet ist eine Vielzahl an Rezepten zu finden, welche eine große Unterstützung beim Kochen und eine Erweiterung des eigenen kulinarischen Horizonts bieten können. Besonders hochwertig und vergleichsweise einfach in der Zubereitung sind die Rezepte auf der Webseite \url{https://www.hensslers-schnelle-nummer.de/}. Problematisch ist jedoch trotzdem, dass die Rezepte meist sehr verschiedene Zutaten benötigen und damit für jedes Rezept einzeln eingekauft werden müsste. Eine Planung mehrerer Rezepte ist für eine ganze Woche erweist sich allerdings ebenfalls als aufwendig, denn es müssen Rezepte mit übereinstimmenden Zutaten gefunden werden.
+
+Die im Rahmen dieses Programmentwurfs entwickelte Software erfüllt den Nutzen, dass Rezepte mit möglichst großer Übereinstimmung gefunden und betrachtet werden können, was die Planung des Wocheneinkaufes erheblich erleichtert.
+
+\section{Funktionalität}
+Die Rezepte werden vollautomatisch aus der Webseite extrahiert und in einer Datenbank persistiert. Dabei werden in der Datenbank bereits vorhandene Rezepte übersprungen, um inkrementelle Aktualisierungen zu ermöglichen. Über eine Kommandozeilenanwendung können die Rezepte einzeln angezeigt, durchsucht und gefiltert werden.
```

Comment:
"in einer Datenbank persistiert" => technische Anforderung => hier sinnvoll?

Reply:
Finde ich schon sinnvoll zu erwähnen, dass es persistent gespeichert wird. Man könnte vllt noch eine Begründung hinzufügen, von wegen "um das Programm auch offline nutzen zu können" oder so.

Reply:
Das finde ich auch sinnvoll, aber ich würde nur erwähnen, dass es persistiert wird, nicht dass es in einer Datenbank passiert

Reply:
Ist korrigiert.